website/public/js/controllers/partyCtrl.js

```diff
@@ -51,6 +51,11 @@ habitrpg.controller("PartyCtrl", ['$rootScope','$scope','Groups','Chat','User','
           Analytics.track({'hitType':'event','eventCategory':'button','eventAction':'click','eventLabel':'Leave Party'});
           $scope.selectedGroup = group;
           $scope.popoverEl = $($event.target);
+
+          if ($scope.popoverEl[0].localName == "span") {
+              $scope.popoverEl = $($scope.popoverEl[0].parentNode);
+          }
```

Comment:
Instead of checking the tag can we check the el has the `.btn` class. I think this would make it a lot more obvious what the point of this block of code is.